### PR TITLE
Bruce/add merchant illustration

### PR DESCRIPTION
**Pull request by bvillaroman**
- added to Seller type to include hero_image_url
- added hero image banner
- fixed the owner panel placement when viewing through mobile

### CODE REVIEW

File: src/components/SellerPage/styles.module.scss [hunk 1]
```diff
@@ -26,11 +26,13 @@
 
 .ownerPanel {
   order: 1;
+  grid-row: 1; /* NEW */
```
Comment (jtmckibb) [inline src/components/SellerPage/styles.module.scss:29]: what's the new comment for?

File: src/components/StoreInfo/StoreInfo.tsx [hunk 2]
```diff
@@ -13,6 +13,17 @@ export const StoreInfo: React.SFC<Props> = ({ seller }) => {
 
   return (
     <section className={classnames(styles.container)}>
+      <div className={styles.merchantIllustrationContainer}>
+        { 
+          seller.hero_image_url && (
+            <img 
+              src={seller.hero_image_url} 
+              alt="merchantIllustration"
```
Comment (jtmckibb) [inline src/components/StoreInfo/StoreInfo.tsx:21]: We could make the alt the seller.name + " Illustration"

File: src/components/StoreInfo/styles.module.scss [hunk 3]
```diff
@@ -1,3 +1,15 @@
+.container {
+  width: 100%;
+}
+.merchantIllustrationContainer {
+  background: #A9122A;
+  width: 100%;
+  height: 12rem;
```
Comment (jtmckibb) [inline src/components/StoreInfo/styles.module.scss:7]: What's rem? I think we should use px

Comment (bvillaroman) [reply src/components/StoreInfo/styles.module.scss:7]: whoops, force of habit

File: src/components/SellerPage/styles.module.scss (hunk 1 above)
Comment (bvillaroman) [reply src/components/SellerPage/styles.module.scss:29]: was a little marker for debugging

**Comment by jtmckibb** [comment]
Looks great tyy!!

**Pull request merged**